about.html

```diff
@@ -12,57 +12,69 @@
 </head>
 
 <body>
-    <div class="container about-page">
+  <header>
+    <nav class="navBar">
+        <img src="img/lambda-black.png">
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="#">Products</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+    </nav>
+    <img src="img/jumbo-about.png">
+</header>
+    <div class="aboutcontainer">
      
-    About
+    <h3 class="topBottom">About</h3>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+    <p class="aboutLine">Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
 
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
+<div class="l">
+    <img src="img/about-plan.png" alt="strategy" class="mid">
+    <h3 class="mid">
+    Strategy</h3>
     
-    How We Work
+    <p class="mid">Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+<a href="###" class="learnMore" class="mid">
+    Learn More</a>
+  </div>
+
```

Comment:
While class names like l and r are not very descriptive (I got it meant left and right) sometimes less is better.  I err on the side of a little bit more descriptive, like right-col, left-col but that is just me.  Good job reusing classes.

Reply:
Try to make class labels more sensible ok sweet